mypy/parse.py

```diff
@@ -5,6 +5,7 @@
 """
 
 import re
+from inspect import cleandoc
```

Comment:
This version of the parser is doomed, we'll remove it as soon as we can use the fast parser on all platforms. Maybe you could skip changing this file and just state that your feature depends on `--fast-parser`?